pages/proof-of-yield/[timestamp]/[strategy].tsx

```diff
@@ -0,0 +1,539 @@
+import { Link } from "../../../types";
+import { useRouter } from "next/router";
+import Head from "next/head";
+import {
+  Button,
+  Footer,
+  GradientButton,
+  Header,
+  Section,
+} from "../../../components";
+import React, { useMemo, useState } from "react";
+import moment, { Moment } from "moment/moment";
+import Error from "../../404";
+import { GetStaticPaths, GetStaticProps, GetStaticPropsContext } from "next";
+import { assetRootPath, fetchAPI, transformLinks } from "../../../utils";
+import { Container } from "../../../components/Container";
+import {
+  strategies,
+  StrategyInfo,
+} from "../../../sections/proof-of-yield/utils/strategies";
+import { ContainerHeader } from "../../../components/ContainerHeader";
+import { ContainerBody } from "../../../components/ContainerBody";
+import { ExternalLinkButton } from "../../../components/ExternalLinkButton";
+import LineBarChart from "../../../components/LineBarChart";
+import { fetchDailyYields } from "../../../queries/fetchDailyYields";
+import { formatEther } from "viem";
+import Image from "next/image";
+import { useQuery } from "react-query";
+import { twMerge } from "tailwind-merge";
+import { TimeUnit } from "chart.js";
+import Tooltip from "../../../components/proof-of-yield/Tooltip";
+import * as dn from "dnum";
```

Comment:
you can destructure imports to leverage tree-shaking [ref](https://github.com/bpierre/dnum?tab=readme-ov-file#tree-shaking)